examples/website/i3s/deckgl/mesh-layer/mesh-layer-vertex.glsl.js

```diff
@@ -27,8 +27,21 @@ out vec4 position_commonspace;
 out vec4 vColor;
 
 void main(void) {
-  // https://github.com/Esri/i3s-spec/blob/master/docs/1.7/geometryUVRegion.cmn.md
-  vec2 uv = fract(texCoords) * (uvRegions.zw - uvRegions.xy) + uvRegions.xy;
+  //------------------------
+  // New code
+  //------------------------
+  #ifdef HAS_UV_REGIONS
+    // https://github.com/Esri/i3s-spec/blob/master/docs/1.7/geometryUVRegion.cmn.md
+    vec2 uv = fract(texCoords) * (uvRegions.zw - uvRegions.xy) + uvRegions.xy;
+  #else
+    vec2 uv = texCoords;
+  #endif
```

Comment:
Hide that conditional logic in a helper function?
```suggestion
applyUVRegion(vec2 uv) {
  #ifdef HAS_UV_REGIONS
    // https://github.com/Esri/i3s-spec/blob/master/docs/1.7/geometryUVRegion.cmn.md
    return fract(texCoords) * (uvRegions.zw - uvRegions.xy) + uvRegions.xy;
  #else
    return = texCoords;
  #endif
}

vec2 uv = applyUVRegion(texCoords, uvRegions);
```

Reply:
Done